Gemfile

```diff
@@ -37,7 +37,9 @@ gem 'colorize'
 
 gem 'kaminari', '~> 1.1'
 
-gem 'aws-sdk-s3'
+gem 'aws-sdk'
```

Comment:
could we not be a bit more specific? We only need S3 and cognito SDKs at this point. This pulls **loads** of stuff

Reply:
Switched to aws-sdk-cognitoidentityprovider